lfucg-exactions/server/plats/views.py

```diff
@@ -74,12 +78,22 @@ def get(self, request, *args, **kwargs):
                 'Current  Exactions',
                 'Current Sewer Exactions',
                 'Current Non-Sewer Exactions',
-                'Payment Total - 1',
-                'Payment Total - 2',
-                'Payment Total - 3',
-                'Payment Total - 4',
-                'Payment Total - 5',
             ])
+        
+            payment_length_per_lot = 0
+            ledger_length_per_lot = 0
+
+            for single_lot in lot_queryset:
+                payment_queryset = Payment.objects.filter(lot_id=single_lot.id)
+                while len(payment_queryset) > payment_length_per_lot:
+                    headers.extend(['Payment Total',])
+                    payment_length_per_lot = payment_length_per_lot + 1
+
+            for single_lot in lot_queryset:
+                ledger_from_queryset = AccountLedger.objects.filter(account_from=single_lot.account)
+                while len(ledger_from_queryset) > ledger_length_per_lot:
+                    headers.extend(['Account Ledger Credits Spent'])
```

Comment:
I'm know I'm being a little picky here, but "Account Ledger Credits Spent" is going to make for a really wide column header, can we reduce this to something less wordy like "Credits Applied" or something?